examples/interactions/commands/application_commands.py

```diff
@@ -59,12 +59,14 @@ async def favorite(
     # and if it matches up to an anime,
     # it responds with a custom response.
     anime = anime.value
-    
+
     match anime:
-        case "Attack on Titan":
+        case 'Attack on Titan':
             await inter.resp.send('It seems like you like Attack on Titan, Nice!')
         case "JoJo's Bizzare Adventure":
-            await inter.resp.send("おにいちゃんありがとう. You like JoJo's Bizzare Adventure. Nice!")
+            await inter.resp.send(
+                "おにいちゃんありがとう. You like JoJo's Bizzare Adventure. Nice!"
+            )
```

Comment:
Reverts out of scope change.
```suggestion
            await inter.resp.send("おにいちゃんありがとう. You like JoJo's Bizzare Adventure. Nice!")
```